version: '3.7'
services:
  dropbox-sync:
    image: urbalurba-dropbox:latest
    build:
      context: .
      dockerfile: Dockerfile
    environment:
      DROPBOX_DIR: /sync
    volumes:
      - ${URBALURBAFILES_VOLUME_NAME:-urbalurba_urbalurbafiles}:/sync
      - dropbox_data:/dropbox
    networks:
      - urbalurba-network
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure

volumes:
  dropbox_data:

networks:
  urbalurba-network:
    external: true
